app/routes/project/posts/post.js

```diff
@@ -32,6 +32,16 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    save(post) {
+      return post.save().catch((error) => {
+        let payloadContainsValidationErrors = error.errors.some((error) => error.status === 422 );
+
+        if (!payloadContainsValidationErrors) {
+          this.controllerFor('project.posts.post').set('error', error);
```

Comment:
@JoshSmith I wouldn't touch this here, since it's really out of the PRs scope and I believe we do the same in other places. We should probably create a separate issue.